templates/default/sssd.conf.erb

```diff
@@ -4,7 +4,8 @@ services = nss, pam<% if node['sssd_ldap']['ldap_autofs'] %>, autofs<% end %><%
 domains = LDAP
 
 [nss]
-filter_users = root,named,avahi,haldaemon,dbus,radiusd,news,nscd
+filter_users = <%= node['sssd_ldap']['filter_users'].join(',') %>
+filter_groups = <%= node['sssd_ldap']['filter_groups'].join(',') %>
```

Comment:
How does this behave if the array is empty and we end up with 'filter_groups = '?